JS/Lesson17/sinan/script.js

```diff
@@ -134,38 +134,84 @@ console.log('----------------------------');
 // Input: ({name: "John", age: 30}, "city", "New York")
 // Output: {name: "John", age: 30, city: "New York"}
 
+function getKeyValue (obj,key,value) {
+if(obj[key]=value) return obj;
```

Comment:
"=" symbol is used in JS for assignment, it should never be used in the condition statement like here. 
What you need to do in this exercise, is add the key value properties to the object:
```
  if (obj[key]) {
   return obj
} else {
  return obj[key] = value;
}
  
```